compact_activity_snapshot.proto

```diff
@@ -262,7 +284,7 @@ message Backend {
     WAIT_EVENT_WAL_SYNC = 965; // WALSync
     WAIT_EVENT_WAL_SYNC_METHOD_ASSIGN = 966; // WALSyncMethodAssign
     WAIT_EVENT_WAL_WRITE = 967; // WALWrite
-    WAIT_EVENT_PROC_SIGNAL_BARRIER = 968; // ProcSignalBarrier
+    WAIT_EVENT_PROC_SIGNAL_BARRIER = 968; // ProcSignalBarrier (moved to IPC category in 13)
```

Comment:
[note for the future] in Postgres 14, 4 new pgstat_get_wait_io events were added. However, this ones were never used and planned to be removed in Postgres 15, therefore I didn't add them here. See below commits for the context:

https://github.com/postgres/postgres/commit/464824323e57dc4b397e8b05854d779908b55304
https://github.com/postgres/postgres/commit/671eb8f34404d24c8f16ae40e94becb38afd93bb
https://github.com/postgres/postgres/commit/a6a0ae127e76da58e0396cd0f21a97066da2115b